models/turbine_models/custom_models/sd3_inference/sd3_mmdit.py

```diff
@@ -249,7 +253,7 @@ class CompiledMmdit(CompiledModule):
             hidden_states_shape,
             encoder_hidden_states_shape,
             pooled_projections_shape,
-            init_batch_dim,
+            (1,),
```

Comment:
is mmdit not working batched?

Reply:
No, this timestep input gets expanded in the model to the batch dimension.